scheduler/crates/sdk/src/user.rs

```diff
@@ -15,6 +15,28 @@ pub struct UpdateUserInput {
 
 pub type CreateUserInput = create_user::RequestBody;
 
+pub struct GetUserFreeBusyInput {
+    pub start_ts: i64,
+    pub end_ts: i64,
+    pub calendar_ids: Option<Vec<ID>>,
+}
+
+impl Into<String> for GetUserFreeBusyInput {
```

Comment:
I would impl "From", instead of "Into". I think `cargo clippy` will also say the same.